lib/experimental/Lists/DataList/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { FC, forwardRef, ReactElement } from "react"
+
+import { IconType } from "@/components/Utilities/Icon"
+import { ItemContainer } from "@/experimental/Lists/DataList/ItemContainer"
+import { ItemWithCopyButton } from "@/experimental/Lists/DataList/ItemWithCopyButton"
+import { LinkItem } from "@/experimental/Lists/DataList/LinkItem"
+
+export type DataListProps = {
+  children: ReactElement<Items>[] | ReactElement<Items>
+  label?: string
+}
+
+type Items =
+  | typeof Item
+  | typeof ItemWithCopyButton
+  | typeof LinkItem
+  | typeof EntityItem
+
+const _DataList = forwardRef<HTMLUListElement, DataListProps>(
+  ({ children, label }, ref) => {
+    return (
+      <ul
+        className="flex min-w-32 max-w-72 flex-col gap-0.5 px-1.5 py-1"
+        ref={ref}
+      >
+        {label && (
+          <p className="px-1.5 text-f1-foreground-secondary">{label}</p>
```

Comment:
A `p` element is not allowed inside a `ul` element [more info in MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul)

![image](https://github.com/user-attachments/assets/6c012426-0244-4908-9c19-e55fea1b51fb)

Probably we'd need to move the `p` outside and add an `aria-labelledby` to indicate what is is labeling.

Reply:
great catch 👍  on it